### ISSUE TITLE
Batch mode triggers github limitations

### ISSUE THREAD

**Issue by guibou** [guibou/krank#17]
Running `krank` in a batched way triggers github limitations:

```
{
  "message": "API rate limit exceeded for 78.194.206.156. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)",
  "documentation_url": "https://developer.github.com/v3/#rate-limiting"
}
```

**Comment by gbataille** [comment]
ahah, I new that was the next step :)

**Comment by guibou** [comment]
Fixed in #25.
